src/SFML.Graphics/Shader.cs

```diff
@@ -498,7 +498,7 @@ public unsafe void SetUniformArray(string name, Glsl.Mat4[] array)
         /// <param name="x">Value to assign</param>
         ///
         ////////////////////////////////////////////////////////////
-        [Obsolete("SetParameter is deprecated, please use the corresponding SetUniform")]
+        [Obsolete("Use SetUniform()")]

```

Comment:
Wonder if we should keep the `the corresponding` since there are matching overloads

Reply:
There are matching overloads that don't specify the type of argument they take, so the end-user would be calling `.SetUniform(whateverTypeOfUniform)` and shouldn't be prompted to look for another method that corresponds to a specific argument type.

Reply:
I guess we can keep it as is